extensions/os_cmd.js

```diff
@@ -8,30 +8,32 @@ const {exec} = require("child_process");
 const {execSync} = require("child_process");
 const {Ticketing} = require(`${CONSTANTS.LIBDIR}/Ticketing.js`);
 
+const _toBuffer = (pid, data) => Buffer.concat([Buffer.from(`[PID:${pid}]`), Buffer.from(data, 'binary')]);
+
 exports.os_cmd = (cmd, stream = false) => {
     if (!CONSTANTS.OBJECT_STORE["ext.os_cmd.ticketing"])
-        CONSTANTS.OBJECT_STORE["ext.os_cmd.ticketing"] = new Ticketing(CONSTANTS.MAX_PROCESSES, "Process pool exhaused, waiting.");
+        CONSTANTS.OBJECT_STORE["ext.os_cmd.ticketing"] = new Ticketing(CONSTANTS.MAX_PROCESSES, "Process pool exhausted, waiting.");
     const ticketing = CONSTANTS.OBJECT_STORE["ext.os_cmd.ticketing"];
 
     CONSTANTS.LOGINFO(`[REQUEST]: ${cmd}`);
 
     return new Promise((resolve, reject) => ticketing.getTicket(_=>{
-        let osProcess = exec(cmd, {maxBuffer: CONSTANTS.MAX_STDIO_BUFFER}, (error, data, stderr) => {
+        let osProcess = exec(cmd, {maxBuffer: CONSTANTS.MAX_STDIO_BUFFER, encoding : 'binary'}, (error, data, stderr) => {
```

Comment:
Small nitpick but we use double quotes as a convention so this should be "binary" not 'binary'.